### ISSUE TITLE
Any way to get client logging messages on failure?

### ISSUE THREAD

**Issue by jjohnstn** [spotify/docker-client#130]
I am using docker-client as the underlying support for Eclipse plug-ins and everything is working well.  When a problem occurs, however, all I am seeing in the DockerException message is that error X has occurred (e.g. 500).  Sometimes, the code can be very vague in meaning.  I have noticed that the client when run from the command line on Linux will issue some nice verbose messages in the lines of: Container xxxxxx cannot be paused as it is not running

Is there some way to get these verbose explanations to pass back to the end-user?

**Comment by rgrunber** [comment]
Just as a follow-up, @jjohnstn found that within the docker-client structure, ContainerInfo (return by inspectContainer), which has ContainerState, also contains a field called "Error", not currently exposed by docker-client that can in some cases hold error messages associated when containers fail in certain ways.

There are probably other error messages that are implemented in the reference implementation of the client, that docker-client (java) will have to implement for itself. For example, in the case of "Container xxxxxx cannot be paused as it is not running", the API might return some error code, and the client manually checks whether the container was in the running state to determine if it should print that error string.

**Comment by jjohnstn** [comment]
I actually found a work-around.  For those messages I don't like, the docker-client is throwing a DockerRequestException.  While the exception's getMessage() returns the request plus return code, there is an added method called message() which has the user message I am looking for.   I have added an extra catch for DockerRequestException and I am seeing much better messages for things such as "executable 'xxxx' not found in $PATH".
